I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 81, in script
    $ actions1 ['Make Bed, Set Alarm, Put Shoes Away']
  File "game/script.rpy", line 81, in <module>
    $ actions1 ['Make Bed, Set Alarm, Put Shoes Away']
NameError: name 'actions1' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 81, in script
    $ actions1 ['Make Bed, Set Alarm, Put Shoes Away']
  File "C:\Users\1023789\Desktop\renpy-8.0.3-sdk\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\1023789\Desktop\renpy-8.0.3-sdk\renpy\python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 81, in <module>
    $ actions1 ['Make Bed, Set Alarm, Put Shoes Away']
NameError: name 'actions1' is not defined

Windows-10-10.0.19044 AMD64
Ren'Py 8.0.3.22090809
project_YouJustWokeUp 1.0
Wed Oct 19 16:36:57 2022
